src/features/saveToList/useSaveObjectToGroup.ts

```diff
@@ -0,0 +1,103 @@
+import { v4 as uuidv4 } from 'uuid';
+
+import { FD } from 'src/features/formData/FormDataWrite';
+import { ALTINN_ROW_ID, DEFAULT_DEBOUNCE_TIMEOUT } from 'src/features/formData/types';
+import { useDataModelBindings } from 'src/features/formData/useDataModelBindings';
+import { useNodeItem } from 'src/utils/layout/useNodeItem';
+import type { IDataModelBindingsForGroupCheckbox } from 'src/layout/Checkboxes/config.generated';
+import type { IDataModelReference } from 'src/layout/common.generated';
+import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+
+type Row = Record<string, string | number | boolean>;
+
+export const useSaveObjectToGroup = (node: LayoutNode<'List' | 'Checkboxes' | 'MultipleSelect'>) => {
+  const bindings = useNodeItem(node, (i) => i.dataModelBindings) as IDataModelBindingsForGroupCheckbox;
+  const checkedBinding = bindings.checked;
+  const checkedBindingSegments = checkedBinding?.field.split('.');
+  const checkedBindingKey = checkedBindingSegments?.pop();
```

Comment:
It's not always given that the `checked` property is one object level deep withing the `group`? We have no such limitations in other data model bindings today, at least. I.e.:

```json
"dataModelBindings": {
  "group": "FormData.People",
  "checked": "FormData.People.Internals.IsChecked"
}
```

Reply:
You should probably look into finding the relative path from within the `group` binding and use something like `dot.pick()` to get values from a row object.